ui/viewport_test.go

```diff
@@ -0,0 +1,48 @@
+package ui
+
+import (
+	"bytes"
+	"context"
+	"github.com/algorandfoundation/hack-tui/api"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/x/exp/teatest"
+	"github.com/oapi-codegen/oapi-codegen/v2/pkg/securityprovider"
+	"testing"
+	"time"
+)
+
+func Test_ViewportViewRender(t *testing.T) {
+	apiToken, err := securityprovider.NewSecurityProviderApiKey("header", "X-Algo-API-Token", "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+	client, err := api.NewClientWithResponses("http://localhost:4001", api.WithRequestEditorFn(apiToken.Intercept))
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Create the Model
+	m, err := MakeViewportViewModel(context.Background(), client)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tm := teatest.NewTestModel(
+		t, m,
+		teatest.WithInitialTermSize(160, 40),
+	)
+
+	// Wait for prompt to exit
+	teatest.WaitFor(
+		t, tm.Output(),
+		func(bts []byte) bool {
+			return bytes.Contains(bts, []byte("Protocol Voting"))
```

Comment:
The viewport should contain everything right? So should we check that the bytes for the status window and, at least, the headers of the account table are also included?

Reply:
Definitely not worth it. The test suite is too experimental to provide good results. Most projects don't even test their tea components.